hack/goreleaser.sh

```diff
@@ -59,11 +59,6 @@ save_images() {
   docker save ${IMAGE_REGISTRY}/${IMAGE_REPOSITORY}:${GORELEASER_CURRENT_TAG}-amd64 > ${IMAGE_SAVE_LOAD_DIR}/${IMAGE_FILE_NAME_PREFIX}-amd64.tar
```

Comment:
In line 53 you still build tests. Also, the test image is built during the release (release.yaml). So I see two options:
- Add `--id botkube` in both places, or
- Remove test build configuration from goreleaser

Both options are fine for me 👍 Generally building the test image won't be necessary anymore, and as we won't use it, it'll be hard to make sure it works. So I think it's better (faster when it comes to CI pipelines) to remove it completely 👍

Reply:
I'll remove building tests all together.